### PR TITLE
Add tree to store copy of dependency's license

### PR DESCRIPTION
**Pull request by kevin-wangzefeng**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

**What this PR does / why we need it**:
Add tree to store copy of dependency's license
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: hack/lib/util.sh [hunk 1]
```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+# Copyright 2014 The Kubernetes Authors.
+#
```
Comment (daixiang0) [inline hack/lib/util.sh:4]: Need sync year here? Not sure about the year in the copyright.

**Comment by daixiang0** [comment]
Seems i cannot add commit here, it has been finished at local.

**Comment by kevin-wangzefeng** [comment]
> Seems i cannot add commit here, it has been finished at local.

You can submit it in a follow-up PR.

**Pull request merged**